p_sets/5/linked_list.rb

```diff
@@ -0,0 +1,48 @@
+# exercises from https://www.sitepoint.com/ruby-interview-questions-linked-lists-and-hash-tables/
+# 1. Print out a linked list given the first node
+
+#### Creating a node class:
+
+class Node
+  attr_accessor :value, :next_node
+
+  def initialize(value, next_node)
+    @value = value
+    @next_node = next_node
+  end
+
+  def to_s
+    current_node = self
+    result = "["
+    until current_node.next_node.nil?
+      result = result + current_node.value.to_s + ", "
+      current_node = current_node.next_node
+    end
+    result + current_node.value.to_s + "]"
+  end
+end
+
+#### Creating a node struct:
+
+Node = Struct.new(:value, :next_node) do
+  def to_s
+    current_node = self
+    result = "["
+    until current_node.next_node.nil?
+      result = result + current_node.value.to_s + ", "
+      current_node = current_node.next_node
+    end
+    result + current_node.value.to_s + "]"
+  end
+end
```

Comment:
I'd suggest deleting whichever of these `Node`s you're not using - less code is better code.